packages/wallet-sdk/src/CoinbaseWalletSDK.ts

```diff
@@ -13,11 +13,12 @@ import { MobileRelay } from './relay/MobileRelay';
 import { WalletLinkRelay } from './relay/WalletLinkRelay';
 import { WalletSDKRelayEventManager } from './relay/WalletSDKRelayEventManager';
 import { getFavicon, isMobileWeb } from './util';
+import { LIB_VERSION } from './version';
 
-const LINK_API_URL = process.env.LINK_API_URL || 'https://www.walletlink.org';
+const LINK_API_URL =
+  (typeof process !== 'undefined' && process.env?.LINK_API_URL) || 'https://www.walletlink.org';
```

Comment:
Lets make a constants file in the wallet-sdk package root and just define the `LINK_API_URL` var as a constant within that file and import that here.

Reply:
:+1: